lib/util.js

```diff
@@ -199,6 +199,13 @@ function deprecate(text) {
   /* eslint-enable */
 }
 
+function deprecateWrap(text, fn, that) {
```

Comment:
I need the original context to rebind it. Without that, I run into trouble in the `apply`:

    function deprecateWrap(text, fn) {
      return function() {
        deprecate(text);
        return fn.apply(/* what? */, arguments);
      };
    }

I could avoid that, but I'd have to do:

    function deprecateWrap(text, fn) {
      return function() {
        deprecate(text);
        // assuming there's no fn with more than 5 args
        var a0 = arguments[0];
        var a1 = arguments[1];
        var a2 = arguments[2];
        var a3 = arguments[3];
        var a4 = arguments[4];
        return fn(a0, a1, a2, a3, a4);
      };
    }